README.md

```diff
@@ -97,6 +97,9 @@ Some of our key features are:
     </li>
     <li>
       Friend-Finding: Discover new friends for both you and your pet! Utilize our advanced friend-finding features to connect with local pet owners!
+      <video width="1280" height="720" controls>
```

Comment:
I don't see this video rendering in the Readme when i open your branch. Looks like you included a .gif not a .webm with the PR. 
Can we also not use the public folder since that is whats used by NextJS. Lets use a /demo folder instead.

Reply:
Also for consistency, let's use the responsive preview with custom device ratio of 1440 x 900